### PR TITLE
Add script to help test pinter

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

File: scripts/bytecode [hunk 1]
```diff
@@ -0,0 +1 @@
+01000000000000004801000000010000006501000000020000006C08000000030000006C01000000040000006F02000000050000002C14000000060000002001000000070000005701000000080000006F180000000900000072010000000A0000006C010000000B00000064010000000C00000021010000000D0000000A
```
Comment (adamhutchings) [inline scripts/bytecode:1]: The output should be in ASCII, not hex

**Review by adamhutchings — CHANGES_REQUESTED** [review]
The bytecode output is in the wrong format, and could you also add the new dependencies to `README` ?

File: scripts/bytecode (hunk 1 above)
Comment (JakeRoggenbuck) [reply scripts/bytecode:1]: Yeah, I realized that while explaining it outload, that's what the "wait one sec" was for lol. I got it I think

**Comment by JakeRoggenbuck** [comment]
Okay, I think we are good to go!

**Review by adamhutchings — APPROVED** [review]
Thanks!

**Pull request merged**